source/_integrations/rfxtrx.markdown

```diff
@@ -25,224 +27,86 @@ There is currently support for the following device types within Home Assistant:
+The `rfxtrx` platform supports Siemens/LightwaveRF and RFY roller shutters that communicate in the frequency range of 433.92 MHz.
 
-### ser2net
+#### Lights
 
-You can host your device on another computer by setting up ser2net and example configuration for ser2net looks like this and then using host/port in your Home Assistant configuration.
+The `rfxtrx` platform support lights that communicate in the frequency range of 433.92 MHz.
 
-```text
-50000:raw:0:/dev/ttyUSB0:38400 8DATABITS NONE 1STOPBIT
-```
+Make sure you trigger a dimming command to get switches detected as lights otherwise, they will show up as switches.
 
-## Setting up your devices
+#### Switches
 
-Once you have set up your [RFXtrx hub](/integrations/rfxtrx/), the easiest way
-to find your binary sensors is to enable automatic add in  `configuration.yaml`:
+The `rfxtrx` platform support switches that communicate in the frequency range of 433.92 MHz.
 
-```yaml
-rfxtrx:
-  automatic_add: true
-  host: 192.168.0.2
-  port: 50000
-```
+#### Sensors
 
-Open your Home Assistant frontend and go to the "states" page.
-Then make sure to trigger your sensor. You should see several new entities
-appear in the *Current entities* list, by looking at the entities attribute, you can look at the last received event, which can be added to the configuration.
+The `rfxtrx` platform support sensors that communicate in the frequency range of 433.92 MHz.
 
-For example: "0913000022670e013b70". Then you should update your configuration to:
+Also, several switches and other devices will also expose sensor entities with battery status as well as the signal level.
 
-```yaml
-rfxtrx:
-  automatic_add: false
-  host: 192.168.0.2
-  port: 50000
-  devices:
-    0913000022670e013b70:
-```
+#### Binary Sensors
 
-### Covers
+The `rfxtrx` platform support binary sensors that communicate in the frequency range of 433.92 MHz. The RFXtrx binary sensor integration provides support for them. Many cheap sensors available on the web today are based on a particular RF chip called *PT-2262*. Depending on the running firmware on the RFXcom box, some of them may be recognized under the X10 protocol, but most of them are recognized under the *Lighting4* protocol. The RFXtrx binary sensor integration provides some special options for them, while other RFXtrx protocols should work too.
 
-The `rfxtrx` platform supports Siemens/LightwaveRF and RFY roller shutters that communicate in the frequency range of 433.92 MHz.
+Customization can be used to set the [class of the device](/integrations/binary_sensor/): [customization](docs/configuration/customizing-devices/)
```

Comment:
Isn't the device class set automatically? We should probably not mention customization as that's a general option available to all integrations.

Reply:
I'll remove it. The device class is set when it's known, but it'll never be a 100% solution because event codes are shared between different device types (there is no standardization). I was trying to limit frustration that this option is gone but generally agree that as it's a general option it should not be mentioned here. The link is also in the PR with the breaking change mentioned so that should be sufficient.